### ISSUE TITLE
Lots of null references

### ISSUE THREAD

**Issue by edent** [pfefferle/wordpress-webmention#342]
Since upgrading to PHP 8.1, I've been getting lots of these errors:

```
PHP Warning:  Attempt to read property "comment_author_url" on null in /blog/wp-content/plugins/webmention/templates/webmention-comment.php on line 7
PHP Warning:  Attempt to read property "comment_parent" on null in /blog/wp-content/plugins/webmention/templates/webmention-comment.php on line 12
PHP Warning:  Attempt to read property "comment_post_ID" on null in /blog/wp-content/plugins/webmention/templates/webmention-comment.php on line 29
PHP Warning:  Attempt to read property "comment_ID" on null in /blog/wp-content/plugins/webmention/templates/webmention-comment.php on line 45
```
https://github.com/pfefferle/wordpress-webmention/blob/3fd2dec6c0e347484be65c15269f83f6ae6651b5/templates/webmention-comment.php#L7
https://github.com/pfefferle/wordpress-webmention/blob/3fd2dec6c0e347484be65c15269f83f6ae6651b5/templates/webmention-comment.php#L12
https://github.com/pfefferle/wordpress-webmention/blob/3fd2dec6c0e347484be65c15269f83f6ae6651b5/templates/webmention-comment.php#L29
https://github.com/pfefferle/wordpress-webmention/blob/3fd2dec6c0e347484be65c15269f83f6ae6651b5/templates/webmention-comment.php#L45

**Comment by edent** [comment]
Closed as 5.0 is released.

**Comment by pfefferle** [comment]
At least one positive thing ;)
